src/data/contributors/Jithendra.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "Jithendra",
+    "skills": ["ML", "java" ,"FullStack"],
+    "cause": "the universe",
+    "image": "Avatar1",
+    "github": "https://github.com/kunal-kushwaha",
+    "twitter": "https://twitter.com/kush_kunal",
+    "linkedin": "https://www.linkedin.com/in/kunal-kushwaha/"
```

Comment:
These are not yours. Please update it with your social handles.

Reply:
ya sure.